botfront/cypress/integration/nlu_models/nlu_models.spec.js

```diff
@@ -1,143 +1,75 @@
-            checkPublishingState(0, 1);
-            
-            cy.get('[data-cy=offline-model]').click();
-            cy.get('[data-cy=confirm-popup]').contains('No').click();
-            cy.get('[data-cy=confirm-popup]').should('not.exist');
-            checkPublishingState(0, 1);
+    it('should NOT be able to delete the default model', function() {
+        cy.visit(`/project/${this.bf_project_id}/settings`);
+        cy.get('.project-default-language .ui > .search').click();
+        cy.get('.project-default-language input').type('English');
+        cy.get('.project-default-language').contains('English').click();
+        cy.get('.project-settings-menu-info').click();
+        cy.contains('Save Changes').click();
 
-            cy.get('[data-cy=offline-model]').click();
-            cy.get('[data-cy=confirm-popup]').contains('Yes').click();
-            cy.get('[data-cy=confirm-popup]').should('not.exist');
-            checkPublishingState(1, 0);
-
-            cy.get('[data-cy=duplicate-button]').click();
-            cy.get('[data-cy=confirm-popup]').contains('Yes').click();
-            checkPublishingState(1, 1);
-
-            cy.get('[data-cy=offline-model]').click();
-            cy.get('[data-cy=confirm-popup]').contains('Yes').click();
-            checkPublishingState(1, 1);
-            cy.get('[data-cy=nlu-model-card]').first().get('[data-cy=offline-model]');
-        
-            cy.get('[data-cy=offline-model]').click();
-            cy.get('[data-cy=confirm-popup]').contains('Yes').click();
-            checkPublishingState(1, 1);
-            cy.get('[data-cy=nlu-model-card]').first().get('[data-cy=online-model]');
-
-            deleteSecondModel();
-            checkPublishingState(1, 0);
-        });
+        // Try to delete the default model
+        cy.visit(`/project/${this.bf_project_id}/nlu/models`);
+        cy.get('.nlu-menu-settings').click();
+        cy.contains('Delete').click();
+        cy.get('.dowload-model-backup-button').click();
+        cy.get('.delete-model-button').should('be.disabled');
     });
 
-    describe('NLU Import', function() {
-        it('Should import a json file with training data', function() {
-            this.dropEvent = {
-                dataTransfer: {
-                    files: [{ path: `${Cypress.config('fixturesFolder')}/nlu_import.json` }],
-                },
-            };
-            this.dropEvent.force = true; // https://github.com/cypress-io/cypress/issues/914
-            cy.visit(`/project/${this.bf_project_id}/nlu/models`);
-            cy.contains('French').click();
-            cy.get(':nth-child(1) > .extra > .basic > .primary').click();
-            cy.get('.nlu-menu-settings').click();
-            cy.contains('Import').click();
-            cy.fixture('nlu_import.json', 'utf8').then((content) => {
-                cy.get('.file-dropzone').upload(content, 'data.json');
-            });
-
-            cy.contains('Import Training Data').click();
```

Comment:
Why is this test removed?

Reply:
I removed it now because in several other tests a createNLUModelWithImport method is called, which does the exact same thing. So I thought it will be redundant to use have it again. I will add it again